### ISSUE TITLE
Sneakers service is triggering errors after ruby 3.1 update

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3305]
Unexpected error PRECONDITION_FAILED - inequivalent arg ‘x-dead-letter-exchange’ for queue ‘catalog_production’ in vhost ‘/’: received none but current is the value ‘catalog_production-retry’ of type ’longstr

The error increases dramatically the volume of the syslog.log file.

**Comment by christinach** [comment]
@kayiwa will check and adjust the file rotation in the operating system so that large logs won't stay for more than a specific amount of days in the system.

**Comment by christinach** [comment]
1. I logged in rabbitmq staging (ticket for documentation https://github.com/pulibrary/orangelight/issues/3309)
2. Checked that there were no messages in catalog_staging queue 
3. Celeted catalog_staging queue
4. Followed the same steps for rabbitmq 🐰 production and deleted catalog_production queue after making sure that there were no messages in this queue. After this step I checked again  the consumers section and it was populated with the consumers IPs.
